zerver/migrations/0561_alter_realm_can_delete_any_message_group.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 5.0.6 on 2024-07-18 15:06
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
I'm not sure if I'm reading this wrong or this should all just be in a single migration. Can you try removing these three migrations and asking django to make it again? Ideally, it should be able to avoid the AlterField part since it'd know of its existence from the start.

Reply:
This is the structure we use for migrations related to settings based on the user groups permission framework. First create a field in `realm` table with `null` value. Then manually create a migration to set a default value and then alter the field to make it `not null`.

Reply:
In that case, I'll mark it as resolved. I'm not experienced with this framework yet.